.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -67,24 +67,24 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-18.04, ubuntu-20.04]
+        os: [ubuntu-20.04, ubuntu-22.04]
     outputs:
-      traffic_light_on_ubuntu_1804: ${{ steps.do_check_on_ubuntu_1804.outputs.light }}
-      traffic_light_on_ubuntu_2004: ${{ steps.do_check_on_ubuntu_2004.outputs.light }}
+      traffic_light_on_ubuntu_1804: ${{ steps.do_check_on_ubuntu_2004.outputs.light }}
```

Comment:
Had better change `traffic_light_on_ubuntu_1804` to `traffic_light_on_ubuntu_2004`